### PR TITLE
test: utils: use updated profile, not original

### PR DESCRIPTION
**Pull request by fromanirh**
UpdateWithRetry wants to get a fresh copy of the profile
to update to avoid conflicts; because of a copy/paste
mistake, it was trying to reapply the original profile,
not the updated copy, thus defeating the purpose of the function.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: test/e2e/performanceprofile/functests/1_performance/hugepages.go [hunk 1]
```diff
@@ -203,6 +204,18 @@ func getCentosPod(nodeName string) *corev1.Pod {
 							MountPath: "/dev/hugepages",
 						},
 					},
+					SecurityContext: &corev1.SecurityContext{
```
Comment (yanirq) [inline test/e2e/performanceprofile/functests/1_performance/hugepages.go:207]: I assume this is a fix for the security context that should land in another PR?

Comment (fromanirh) [reply test/e2e/performanceprofile/functests/1_performance/hugepages.go:207]: yes, turns out we need fixes here: https://github.com/openshift/cluster-node-tuning-operator/pull/405

**Review by MarSik — APPROVED** [review]
/lgtm

**Comment by fromanirh** [comment]
/retitle OCPBUGS-246: test: utils: use updated profile, not original
